modules/gateway-task/variables.tf

```diff
@@ -0,0 +1,240 @@
+variable "family" {
+  description = "Task definition family (https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#family). This is used by default as the Consul service name if `consul_service_name` is not provided."
+  type        = string
+}
+
+variable "consul_service_name" {
+  description = "The name the gateway service will be registered as in Consul. Defaults to the Task family name. Always suffixed with the gateway kind."
+  type        = string
+  default     = ""
+}
+
+variable "consul_service_tags" {
+  description = "A list of tags included in the Consul service registration."
+  type        = list(string)
+  default     = []
+}
+
+variable "consul_service_meta" {
+  description = "A map of metadata that will be used for the Consul service registration"
+  type        = map(string)
+  default     = {}
+}
+
+variable "consul_namespace" {
```

Comment:
I believe setting this to anything but `default` breaks things https://www.consul.io/docs/connect/config-entries/mesh#namespace